src/Bridge/Symfony/Container/Modifier/Builder/Symfony63PlusBuilder.php

```diff
@@ -159,7 +159,7 @@ private function generateOverridenLoad(ReflectionClass $reflContainer): string
     private static function generateOverrideOriginalContainerLoad(): string
     {
         return <<<'EOF'
-                protected function load(string $file)
+                protected function load(string $file): mixed
```

Comment:
here are some old classes sf 54 and  63, but for sf ^7.0 return type was missing (just hotfix :))